lib/metrics.rb

```diff
@@ -5,8 +5,8 @@ def content_change_emails(content_change, count)
       count("content_change_emails.document_type.#{content_change.document_type}.immediate", count)
     end
 
-    def unsubscribed(reason)
-      increment("unsubscribed.reason.#{reason}")
+    def unsubscribed(reason, count = 1)
+      count.times { increment("unsubscribed.reason.#{reason}") }
```

Comment:
It'd be more idiomatic to use `count("unsubscribed.reason.#{reason}", count)` here rather than a loop.

It'll do the same thing: https://github.com/reinh/statsd/blob/3863a1a341d21b3c81a0ab9aebc6bd901da45323/lib/statsd.rb#L351-L371